torch_xla/distributed/xla_backend.py

```diff
@@ -41,6 +42,14 @@ def __init__(self, prefix_store, rank, size, timeout):
     self.prefix_store = prefix_store  # reserved for future use.
     self.timeout = timeout
     self._mesh = []
+    # Initialize xrt neuron environment
+    # Passes in the store created by torch.distributed to avoid
+    # creating two TCP stores. We only want to call this
+    # when the user is using torchrun and not xmp.spawn()
+    # or some other flow.
+    if os.getenv('TORCHELASTIC_RUN_ID') != None:
```

Comment:
Is `TORCHELASTIC_RUN_ID` ever set by anything other than torchrun?

Reply:
No, its only specific to torchrun.